### DISCUSSION TITLE
Xplane 12 just crashes when I have autoortho running in the background.

### DISCUSSION THREAD

**Discussion by zippy77777** [kubilus1/autoortho#401] [General]
Hi all
Everything has been working perfectly until today. I don't recall making any changes but xplane 12 will not run with autoortho. Take it out it runs fine. I am dam sure its all installed correctly but I just cannot get it to work. Please help.

![Screenshot 2023-10-01 165121](https://github.com/kubilus1/autoortho/assets/33657120/7bde9bcb-9c07-42f7-b83b-3a4bfd4da697)

**Comment by zippy77777** [comment]
ANYONE PLZ! it's driving me insane.................

**Comment by zippy77777** [comment]
I guess its back to Ortho4XP until a solution is found! :(

**Comment by crberry50** [comment]
getting same issue, Here is what x-plane's log file contains ... maybe some data corruptions for this file ?  How to restore without a complete reload if possible?  Thank you
Edited:  The log file below was from yesterday.   I just started up again today and x-plane started without a problem and did not log this error.  However... After quitting x-plane and starting up again, I once again did get the error.  Why the inconsistency?  (2nd log file is below labeled 'Today' ... 
Additional Edit:   I was able to shutdown and start up x-plane and autoortho with both success and failure today .. again.  It seems like this is a timing issue of some sort.  Question is it best to leave autoortho running for a long time before starting x-plane so that it gathers every possible file it needs to use? ... I am on 0.6.0-1 btw....  Thank you.
====  Yesterday ====
0:04:15.858 E/SCN: ***Error with scenery file "+39-075.dsf" (io_dsf.cpp:776.)***
0:04:15.858 E/SCN: The terrain file 'terrain/24896_19200_BI16.ter' could not be loaded.
0:04:15.858 E/SCN: 
0:04:15.858 E/SYS: MACIBM_alert: Error loading the scenery package: Custom Scenery/z_ao_na/
0:04:15.858 E/SYS: MACIBM_alert: X-Plane cannot run with this package installed. You may need to remove the custom scenery package.
0:04:15.858 E/SYS: MACIBM_alert: See the Log.txt file for detailed error information.
0:04:15.858 E/SYS: MACIBM_alert: http://lookup.x-plane.com/_lookup_12_/broken_custom_scenery.html
0:04:15.858 E/SYS: MACIBM_alert: C:/GitLab-Runner/builds/2dtum6M9/0/x-plane/design/source_code/engine/REN/io_dsf.cpp:776
0:04:15.858 E/SCN: ***Error with scenery file "+39-075.dsf" (io_dsf.cpp:776.)***
0:04:15.858 E/SCN: The terrain file 'terrain/24896_19200_BI16.ter' could not be loaded.
0:04:15.858 E/SCN: 
0:04:15.858 E/SYS: MACIBM_alert: Error loading the scenery package: Custom Scenery/z_ao_na/
0:04:15.858 E/SYS: MACIBM_alert: X-Plane cannot run with this package installed. You may need to remove the custom scenery package.
0:04:15.858 E/SYS: MACIBM_alert: See the Log.txt file for detailed error information.
0:04:15.858 E/SYS: MACIBM_alert: http://lookup.x-plane.com/_lookup_12_/broken_custom_scenery.html
0:04:15.858 E/SYS: MACIBM_alert: C:/GitLab-Runner/builds/2dtum6M9/0/x-plane/design/source_code/engine/REN/io_dsf.cpp:776
0:04:15.858 E/SCN: ***Error with scenery file "+39-075.dsf" (io_dsf.cpp:776.)***
0:04:15.858 E/SCN: The terrain file 'terrain/24896_19200_BI16.ter' could not be loaded.
0:04:15.858 E/SCN: 
0:04:15.858 E/SYS: MACIBM_alert: Error loading the scenery package: Custom Scenery/z_ao_na/
0:04:15.858 E/SYS: MACIBM_alert: X-Plane cannot run with this package installed. You may need to remove the custom scenery package.
0:04:15.858 E/SYS: MACIBM_alert: See the Log.txt file for detailed error information.
0:04:15.858 E/SYS: MACIBM_alert: http://lookup.x-plane.com/_lookup_12_/broken_custom_scenery.html
0:04:15.858 E/SYS: MACIBM_alert: C:/GitLab-Runner/builds/2dtum6M9/0/x-plane/design/source_code/engine/REN/io_dsf.cpp:776
0:05:43.482 W/TEX: Resources/default scenery/1000 world terrain/textures12/apt/apt_ground2_NML.dds compressed texture used as material source
0:05:43.545 W/TEX: Resources/default scenery/1000 world terrain/textures12/apt/apt_ground1_NML.dds compressed texture used as material source
0:05:53.087 I/SCN: DSF load time: 22308020 for file Custom Scenery/z_ao_na/Earth nav data/+40-080/+41-073.dsf (1381555 tris, 7200 skipped for -504.9 m^2)
0:05:53.087 D/SCN: Generating detailed DEM for 'Custom Scenery/z_ao_na/Earth nav data/+40-080/+41-073.dsf'
0:06:01.142 E/SYS: +-------------------------------------------------------------------------------
0:06:01.142 E/SYS: | Error loading the scenery package: Custom Scenery/z_ao_na/
0:06:01.142 E/SYS: | X-Plane cannot run with this package installed. You may need to remove the custom scenery package.
0:06:01.142 E/SYS: | See the Log.txt file for detailed error information.
0:06:01.142 E/SYS: | http://lookup.x-plane.com/_lookup_12_/broken_custom_scenery.html
0:06:01.142 E/SYS: | (io_dsf.cpp:776)
0:06:01.142 E/SYS: +-------------------------------------------------------------------------------
0:06:01.142 D/STM: Transitioning from state state_NormalLoading to state_ShuttingDown

==== Today
0:00:21.328 D/SCN: Generating detailed DEM for 'Custom Scenery/z_ao_na/Earth nav data/+30-120/+33-117.dsf'
0:00:21.328 E/SCN: ***Error with scenery file "+34-119.dsf" (io_dsf.cpp:776.)***
0:00:21.328 E/SCN: The terrain file 'terrain/26112_11264_BI16.ter' could not be loaded.
0:00:21.328 E/SCN: 
0:00:21.328 E/SYS: MACIBM_alert: Error loading the scenery package: Custom Scenery/z_ao_na/
0:00:21.328 E/SYS: MACIBM_alert: X-Plane cannot run with this package installed. You may need to remove the custom scenery package.
0:00:21.328 E/SYS: MACIBM_alert: See the Log.txt file for detailed error information.
0:00:21.328 E/SYS: MACIBM_alert: http://lookup.x-plane.com/_lookup_12_/broken_custom_scenery.html
0:00:21.328 E/SYS: MACIBM_alert: C:/GitLab-Runner/builds/2dtum6M9/0/x-plane/design/source_code/engine/REN/io_dsf.cpp:776
0:00:21.328 E/SCN: ***Error with scenery file "+34-119.dsf" (io_dsf.cpp:776.)***
0:00:21.328 E/SCN: The terrain file 'terrain/26112_11264_BI16.ter' could not be loaded.
0:00:21.328 E/SCN: 
0:00:21.328 E/SYS: MACIBM_alert: Error loading the scenery package: Custom Scenery/z_ao_na/
0:00:21.328 E/SYS: MACIBM_alert: X-Plane cannot run with this package installed. You may need to remove the custom scenery package.
0:00:21.328 E/SYS: MACIBM_alert: See the Log.txt file for detailed error information.
0:00:21.328 E/SYS: MACIBM_alert: http://lookup.x-plane.com/_lookup_12_/broken_custom_scenery.html
0:00:21.328 E/SYS: MACIBM_alert: C:/GitLab-Runner/builds/2dtum6M9/0/x-plane/design/source_code/engine/REN/io_dsf.cpp:776
0:01:26.772 W/TEX: Resources/default scenery/1000 world terrain/textures12/apt/apt_ground1_NML.dds compressed texture used as material source
0:01:43.420 E/SYS: +-------------------------------------------------------------------------------
0:01:43.420 E/SYS: | Error loading the scenery package: Custom Scenery/z_ao_na/
0:01:43.420 E/SYS: | X-Plane cannot run with this package installed. You may need to remove the custom scenery package.
0:01:43.420 E/SYS: | See the Log.txt file for detailed error information.
0:01:43.420 E/SYS: | http://lookup.x-plane.com/_lookup_12_/broken_custom_scenery.html
0:01:43.420 E/SYS: | (io_dsf.cpp:776)
0:01:43.420 E/SYS: +-------------------------------------------------------------------------------

**Comment by SammyBeen** [comment]
Yes. This is happening for me too running XPlane 12 and AO 0.6.0-1  :-(

**Comment by marloscaida** [comment]
Please, read this:
https://github.com/kubilus1/autoortho/discussions/395

**Comment by zippy77777** [comment]
I am all sorted now thx

**Comment by zippy77777** [comment]
I have managed to get it working after 2 days of trying. Dokan no longer works and winFSP does but they don't work together. Delete Dokan. If you use SAM, move the libraries into custom scenery folder manually (settings in xplane) and delete the rest of SAM from the custom scenery folder. Create symbolic links to the remaining library files. This will stop the AO files keep moving to the top of the scenery_packs.ini folder. It took a while experimenting but this eventually worked for me

**Reply by Jeff2122** [reply]
I don't yet understand why Dokan is a problem for some but not others. I am glad you got it working though. 
I have both Dokan and WinFSP installed, but I use Dokan. I am not even sure v0.5.1-3 will work with WinFSP. 
I have tried both with v0.6.x and both work, but v0.5 doesn't have the switch in the .autoortho file to use WinFSP.

**Comment by zippy77777** [comment]
When both Dokan and winFSP installed, it does not work. I am a non steam user with xplane 12.  I am running 0.6.1 the latest and doing everything I have mentioned works for me. I can use both true and False.

**Comment by rtc79clm** [comment]
I am getting the same thing. I have changed the wait time and the thread = true. However, it still crashes, when I try with WinFSP it messes with scenery_pack and the ortho files are never in the correct place. I also use X Organizer. I currently am using 0.5.1-3
